app/controllers/api/accounts_controller.rb

```diff
@@ -12,7 +12,7 @@ def resources
   end
 
   def resources_base
-    user.try(:accounts) || Account
+    user.try(:accounts) || super
```

Comment:
This was an extant bug. We always want this method to return an `ActiveRecord::Relation`.